### PR TITLE
Merge to next for further development

### PR DESCRIPTION
**Pull request by raineio**

### CODE REVIEW

File: .github/workflows/action.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+name: CI
+
+on:
+  push:
+    branches: 
+      - "dev/knex/pgsql"
```
Comment (wolfcomp) [inline .github/workflows/action.yml:6]: Should be set to `next`

File: .github/workflows/action.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+name: CI
+
+on:
+  push:
+    branches: 
+      - "dev/knex/pgsql" 
+      
+    paths-ignore:
```
Comment (wolfcomp) [inline .github/workflows/action.yml:8]: This isn't a thing in the CI documentation check [here](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#push)

File: .github/workflows/lint.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+on:
+    push:
+        branches:
+            - main
+            - dev/knex/pgsql
+    
+    pull_request:
+      branches:
+        - main
+        - dev/knex/pgsql
```
Comment (wolfcomp) [inline .github/workflows/lint.yml:12]: This should be set to `next`

File: .gitignore [hunk 4]
```diff
@@ -103,11 +103,4 @@ dist
...
 
-data
-public
-package-lock.json
-swagger.json
-addChallengesToDb.js
-addToDb.js
-dataStructures.js
-index.html
+*.lock # ??????
```
Comment (wolfcomp) [inline .gitignore:106]: Remove this line `yarn.lock` should always be pushed up

File: src/db.ts [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    pool: {
+        min: 2,
+        max: 10,
+    },
+});
+
+export default db;
+
+declare module "knex/types/tables.js" {
+    interface Tables {
```
Comment (wolfcomp) [inline src/db.ts:23]: Please add something to silence eslint for this one line

**Review by wolfcomp — CHANGES_REQUESTED** [review]
Just some things that should be changed before it gets merged.

**Review by wolfcomp — APPROVED** [review]

File: src/db.ts (hunk 5 above)
Comment (raineio) [reply src/db.ts:24]: [afaa543](https://github.com/Saber-Quest/SaberQuest-Backend/pull/5/commits/afaa543cba9d05e42d6e85c719f37da2ab503b2f)

File: .github/workflows/lint.yml (hunk 3 above)
Comment (raineio) [reply .github/workflows/lint.yml]: Fixed in [afaa543](https://github.com/Saber-Quest/SaberQuest-Backend/pull/5/commits/afaa543cba9d05e42d6e85c719f37da2ab503b2f)

File: .github/workflows/action.yml (hunk 2 above)
Comment (raineio) [reply .github/workflows/action.yml]: Fixed in [afaa543](https://github.com/Saber-Quest/SaberQuest-Backend/pull/5/commits/afaa543cba9d05e42d6e85c719f37da2ab503b2f)

File: .github/workflows/action.yml (hunk 1 above)
Comment (raineio) [reply .github/workflows/action.yml]: Fixed in [afaa543](https://github.com/Saber-Quest/SaberQuest-Backend/pull/5/commits/afaa543cba9d05e42d6e85c719f37da2ab503b2f)

**Pull request merged**